core/crud_component.cxx

```diff
@@ -36,8 +36,137 @@
 #include <asio/io_context.hpp>
 #include <asio/steady_timer.hpp>
 
+#include <tl/expected.hpp>
+
 namespace couchbase::core
 {
+std::pair<std::vector<std::byte>, std::error_code>
+serialize_range_scan_create_options(const range_scan_create_options& options)
+{
+    tao::json::value body{};
+    if (options.without_content) {
```

Comment:
n.b. in latest meeting this was changed to ids_only.  It hasn't hit the RFC as a formal revision yet but I'm assuming it'll go through.

Reply:
fixed